docs/reference/installation.rst

```diff
@@ -138,16 +138,7 @@ SonataAdminBundle Configuration
 SonataBlockBundle Configuration
 -------------------------------
 
-.. code-block:: yaml
-
-    # config/packages/sonata_block.yaml
-
-    sonata_block:
-        context_manager: sonata.page.block.context_manager
-
-.. note::
-
-    Please you need to use the context ``sonata_page_bundle`` in the SonataBlockBundle to add block into a Page.
+You need to use the context ``sonata_page_bundle`` in the SonataBlockBundle to add block into a Page.
```

Comment:
Do you know what does this means?

Reply:
No.

I saw that in tests we have
```
sonata_block:
    http_cache: false
    blocks:
        sonata.admin.block.admin_list:
            contexts: [admin]
    default_contexts: [sonata_page_bundle]
```
But i can't say it's related ; so I didn't try to change the doc